src/pages/data-reporting/reports-api/v3.4/index.mdx

```diff
@@ -430,6 +430,63 @@ https://api.livechatinc.com/v3.4/reports/chats/ranking \
 </Code>
 </Section>
 
+<Section>
+<Text>
+
+### Engagement
+
+Shows the distribution of chats based on engagement during the specified period.
```

Comment:
```suggestion
Shows the distribution of chats based on [engagement](https://www.livechat.com/help/manage-team-reports/) during the specified period.
```

Reply:
I'm not sure if readers will immediately know what we mean by chat engagement, so this article might be helpful.